README.md

```diff
@@ -28,6 +28,12 @@ Using npm:
 
     npm install --save react-stripe-elements
 
+Using UMD build (exports a global `ReactStripeElements` object):
+
+```html
+<script src="https://unpkg.com/react-stripe-elements@latest/dist/react-stripe-elements.min.js"></script>
+```
+
 Then, load Stripe.js in your application:
```

Comment:
Do you mind making this line and the "First, install..." lines headers (`####` probably), since there's now a lot of content under the first step?